bin/release-helper.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 VERSION_FILE=${VERSION_FILE-*/__init__.py}
-DEPENDENCY_FILE=${DEPENDENCY_FILE-setup.cfg}
+DEPENDENCY_FILE=${DEPENDENCY_FILE-pyproject.toml}
```

Comment:
q: Have you tested that the usages of this variable also work for the `pyproject.toml` (and not only for `*.cfg` files)?